src/Farmer/Arm/ContainerRegistry.fs

```diff
@@ -0,0 +1,24 @@
+[<AutoOpen>]
+module Farmer.Arm.ContainerRegistry
+
+open Farmer
+
+type Registries =
```

Comment:
`Registry` maybe?

Reply:
Again, this is based on the official ARM namespace.